simple_network_sim/loaders.py

```diff
@@ -154,41 +227,59 @@ def __init__(self, a, b=None):
         assert self._lower >= 0, f"No one is {self._lower} years old"
 
     def __contains__(self, age):
-        """Returns true if age is inside this age range."""
+        """Return true if age is inside this age range."""
         if age < self._lower:
             return False
         return age < self._upper
 
     def __str__(self):
+        """Return a string representation of the current AgeRange instance."""
         if self._upper == AgeRange.MAX_AGE:
             return f"{self._lower}+"
         return f"[{self._lower},{self._upper})"
 
     def __eq__(self, other):
+        """Return true if "other" is the same as the current AgeRange instance."""
         return self._lower == other._lower and self._upper == other._upper
 
     def __neq__(self, other):
+        """Return true if "other" is not the same as the current AgeRange instance."""
         return not self == other
 
     def __hash__(self):
+        """Return a hash of the current AgeRange instance."""
         return hash((self._lower, self._upper))
 
 
 class MixingRow:
-    """A mixing row is one row of a mixing table. This is a helper class. A row
+    """One row of a mixing table.
+    
+    This is a helper class. A row
```

Comment:
even though it's more change, I think we should probably reformat these lines anyway

Reply:
Resolved by 382b7c5867da2ad3ef5fefc5d3f9bf15296f1b86. I think.